dm/unit/unit.go

```diff
@@ -65,7 +63,7 @@ type Unit interface {
 // we can refine to add error scope field if needed
 func NewProcessError(err error) *pb.ProcessError {
 	result := &pb.ProcessError{
-		Msg: errors.ErrorStack(err),
+		Msg: terror.Message(err),
```

Comment:
```suggestion
		Msg: "",
```

Reply:
After a discussion we think it's better to set empty Msg here and return `result.Error` to users.
But we set `result.Error` to nil in https://github.com/pingcap/dm/blob/master/dm/worker/status.go#L206. Could you help us to remove this line?
What's more, there are some unit tests and code (for example: https://github.com/pingcap/dm/blob/master/checker/cmd.go#L73) using `result.Msg`, can you help us refine these code?

Reply:
ok, let me see